tests/scripts/eggs4testing/build-nspkg-tests.py

```diff
@@ -79,66 +81,39 @@ def make_package(pkgname, namespace_packages, modules, zip_safe=False,
 # collection of packages to be installed using
 #   PYTHONPATH=. python setup.py install --install-lib .
 # This will keep the __init__files of the namespace-packages.
-make_package('nspkg1-aaa',
-             ['nspkg1'],
-             ['nspkg1/aaa/__init__.py'])
-make_package('nspkg1-bbb',
-             ['nspkg1', 'nspkg1.bbb'],
-             ['nspkg1/bbb/zzz/__init__.py'],
-             zip_safe=True)
-make_package('nspkg1-ccc',
-             ['nspkg1'],
-             ['nspkg1/ccc.py'])
-make_package('nspkg1-empty',
-             ['nspkg1'],
-             [],
-             zip_safe=True)
-
+make_package('nspkg1-aaa', ['nspkg1'], ['nspkg1/aaa/__init__.py'])
```

Comment:
The previous formatting structure should be preserved (but with first argument on new line), especially because we now end up with inconsistent formatting.